src/components/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import { Link } from 'react-router-dom';
+
+const Navbar = () => {
+    return (
+        <>
+            <header className='header'>
+                <div className="container dFlex">
+                    <div className="categories dFlex">
+                        <span className='fa-solid fa-border-all'></span>
+                        <h4>
+                            Categories <i className='fa fa-chevron-down'></i>
+                        </h4>
+                    </div>
+                    <div className="navlink">
+                        <ul className="nav">
+                            <li>
+                                <Link to="./">HOME</Link>
+                            </li>
+                            <li>
+                                <Link to="./">SERVICES</Link>
+                            </li>
+                            <li>
+                                <Link to="./store">store</Link>
+                            </li>
+                            <li>
+                                <Link to="./contact">contact</Link>
```

Comment:
awesome link work! not sure if it's intentional, but services doesn't like to anything